File: pset6/pset6/mario.py
import cs50

height = -1
s = "#"

while height > 23 or height < 0:
    print("Height: ", end="")
    height = cs50.get_int()
    
for i in range(1, height + 1):
    print("{}{}{}{}{}".format(
        (height-i) * " ", i*s, "  ",  #left blocks and two space 
        i*s, (height-i) * " ")        #right blocks
        )

File: mario.c
#include <stdio.h>
#include <cs50.h>
void oneline(int line);

int main(void)
{
    printf("Height: ");
    int height = get_int();
    while (height > 23 || height < 0 )
    {
        printf("Height: ");
        height = get_int();
    }
    for (int i = 1; i <= height; i++)
    {
        for (int j = 0; j < height - i; j++)
        {
        printf(" ");
        }    
        oneline(i);
        printf("  ");
        oneline(i);
        printf("\n");
    }
    
}

void oneline(int line)
{
    for (int i = 0; i < line; i++)
    {
        printf("#");
    }
}